src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -0,0 +1,74 @@
+package frc.team2412.robot.subsystem;
+
+import org.frcteam2910.common.math.Rotation2;
+
+import static frc.team2412.robot.subsystem.TargetLocalizer.LocalizerConstants.*;
+
+public class TargetLocalizer {
+    public static class LocalizerConstants {
+        public static final double TURRET_OFFSET = 0;
+        // TODO tune these more
+        public static final double TURRET_LATERAL_FF = 4.8, TURRET_ANGULAR_FF = 4.8, TURRET_DEPTH_FF = 0,
+                LATERAL_DEPTH_COMPENSATION = 0;
+    }
+
+    private final DrivebaseSubsystem drivebaseSubsystem;
+    private final ShooterSubsystem shooterSubsystem;
+    private final ShooterVisionSubsystem shooterVisionSubsystem;
+
+    public TargetLocalizer(DrivebaseSubsystem drivebase, ShooterSubsystem shooter, ShooterVisionSubsystem vision) {
+        drivebaseSubsystem = drivebase;
+        shooterSubsystem = shooter;
+        shooterVisionSubsystem = vision;
+    }
+
+    public double getDistance() {
+        return hasTarget() ? shooterVisionSubsystem.getDistance() + shooterSubsystem.getDistanceBias() : 0;
```

Comment:
Should the distance bias go in `getDistance()` or in `getAdjustedDistance()`?

Reply:
bias is something else internal